### PR TITLE
Update merchants.yml

### PR DESCRIPTION
**Pull request by wintermut9**
Hi, 

Could you kindly add NOWNodes to Tools section? The service provides access to Monero Explorer and Node via API, description can be found here: https://documenter.getpostman.com/view/11776685/SzzkdxDM?version=latest

Issue: https://github.com/monero-project/monero-site/issues/1130

### CODE REVIEW

File: _data/merchants.yml [hunk 1]
```diff
@@ -179,6 +179,8 @@
       url: https://moneroaddress.org/
     - name: Ownbit
       url: https://ownbit.io
+    - name: NOWNodes: Explorer & Node APIs
```
Comment (erciccione) [inline _data/merchants.yml:182]:
```suggestion
    - name: NOWNodes - Explorer & Node APIs
```

Comment (wintermut9) [reply _data/merchants.yml:182]: Hi! Added https://github.com/monero-project/monero-site/issues/1130

Comment (erciccione) [reply _data/merchants.yml:182]: Perfect. Now please add the changes i suggested. This PR is broken at the moment.

**Comment by luigi1111** [comment]
Squashing on Github doesn't create a merge commit AFAIK.

**Pull request merged**